v1.0/Specification.md

```diff
@@ -332,7 +332,7 @@ Table with terms used in the specification.
 | Term | Description |
 | --- | --- |
 | **Archival Information Package (AIP)** | *From OAIS:* An Information Package, consisting of the Content Information and the associated Preservation Description Information (PDI), which is preserved within an OAIS. |
-| **Artefact** | An artefact is a data object in a spreadsheet which may be considered insignificant for preservation and future reuse of the spreadsheet, but removing the object may not be considered neither necessary nor worthwhile. Therefore, the data object may remain in the spreadsheet and considered unnecessary for rendering or future migration. |
+| **Artefact** | An artefact is an object in a spreadsheet which is not a significant property and hence not important for future reuse of the spreadsheet, but removing the object may not be considered neither necessary nor worthwhile. Therefore, the data object may remain in the spreadsheet and considered unnecessary for rendering or future migration. |
```

Comment:
An artefact is a spreadsheet property not considered significant for long-term preservation/reuse. While preserving artefacts is unnecessary, removing them may be considered a preservation risk. It is safe to ignore artefacts when migrating between formats/representations.

Reply:
I implemented this with very slight modifications. Can you approve?